data-pipeline/src/pipeline/main.py

```diff
@@ -248,33 +248,38 @@ def pre_process_bfr(set_type, year):
         raw_container, f"{set_type}/{year}/BFR_3Y_raw.csv", encoding="unicode-escape"
     )
 
-    bfr_sofa, bfr_3y, bfr, bfr_metrics = build_bfr_data(bfr_sofa, bfr_3y)
+    academies_y2 = prepare_data(
+        pd.read_parquet(
+            get_blob("pre-processed", f"{set_type}/{int(year)-2}/academies.parquet")
```

Comment:
We've agreed (as per a comment elsewhere) that `year` will be passed in as an `int` upstream so this should be safe now.